README.md

```diff
@@ -25,3 +25,6 @@ and configuring your .env config file with your Paypal API keys.
 2. Run `npm install`
 3. Run `npm run dev`
 4. Navigate to `http://localhost:8080/`
+
+### PayPal Codespaces Links
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/paypal-examples/sofort)
```

Comment:
let's test this. somehow it did not show options to add client id/secret while creating codespace.